ballerina/tests/08-procedure-call-test.bal

```diff
@@ -79,6 +80,25 @@ isolated function beforeProcCallFunc() returns sql:Error? {
     result = check oracledbClient->execute( `INSERT INTO CallComplexTypes (id, col_xml)
         VALUES(1, XMLType('<key>value</key>'))`);
 
+    result = check dropTableIfExists("CallDateTimeTypes", oracledbClient);
+    result = check oracledbClient->execute(`ALTER SESSION SET NLS_DATE_FORMAT='DD-MON-YYYY HH:MI:SS AM'`);
+    result = check oracledbClient->execute(`ALTER session set NLS_TIMESTAMP_TZ_FORMAT = 'DD-MON-YYYY HH:MI:SS AM TZR'`);
+    result = check oracledbClient->execute(`CREATE TABLE CallDateTimeTypes(
+        id NUMBER,
+        col_date DATE,
+        col_date_only DATE,
+        col_timestamp TIMESTAMP,
+        col_timestamptz TIMESTAMP WITH TIME ZONE,
+        col_interval_year_to_month INTERVAL YEAR TO MONTH,
+        col_interval_day_to_second INTERVAL DAY TO SECOND,
+        PRIMARY KEY(id)
+        )`
```

Comment:
We cant have initialisation scripts mixed with the tests, lets move the ones to separate file and reuse much of the tabled if needed

Reply:
All table initialization will be move to initialized scripts  which will run at docker container startups. will address this concern in a separate PR.  https://github.com/ballerina-platform/ballerina-standard-library/issues/1729